_connect-files/api/objects/form-properties/sources/databases/responsys-object.md

```diff
@@ -0,0 +1,44 @@
+---
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-responsys-object"
+
+title: "Responsys Source Form Property"
+api-type: "responsys"
+display-name: "Responsys"
+
+source-type: "database"
```

Comment:
Actually, looking at the preview. It seems like that gives the awesome table at the top? I think it's really nice to have that. The term database could be confusing since responsys isn't a database, but is an SSH connection. Though I think that's a really minor concern right now, since this is the only one that fits that bill.

Reply:
Agreed! I think we should table this for a later date, especially if we build another integration that fits into this category. Responsys can live with the databases for now.